src/einsteinpy/bodies/body.py

```diff
@@ -0,0 +1,43 @@
+import astropy.constants as const
+import astropy.units as u
+import numpy as np
+
+
+class Body:
+    """
+    Class for making a body.
+
+    If the parameters are not passed, they are taken as the parameters of Sun.
+    The default body is Sun.
+
+    Parameters
+    ----------
+    mass - This is the mass of the body.
+
+    pos_vec - This is the position vector of the body. 
+              It is a numpy array of 3 elements with their units
+
+    vel_vec - This is the velocity vector of the body. 
+              It is a numpy array of 3 elements with their units
+
+    is_attractor - This boolean paramter tells if the body is attractor or not.
+
+    desc - This is the description about the body where the 
+           user can describe the object. It defaults to "Empty" string.
+
+    """
+
+    def __init__(
+        self,
+        mass=const.M_sun.value * u.kg,
```

Comment:
This should not default to sun

Reply:
Okay. In case the user doesn't enter the mass, what should the mass default to?
Should it raise an error something along the lines of "mass must be provided" or default it to 0?